deploy/apim/api-management/groups/ApiMessageWriteDryRun/configuration.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "ApiMessageWriteDryRun",
```

Comment:
This group has been renamed to `ApiLimitedMessageWrite`

Reply:
yes, we must download the final configuration from the cdc account once is stable